src/e2e_test/perf_tool/pkg/cluster/local/local.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package local
+
+import (
+	"context"
+	"errors"
+
+	"px.dev/pixie/src/e2e_test/perf_tool/experimentpb"
+	"px.dev/pixie/src/e2e_test/perf_tool/pkg/cluster"
+	"px.dev/pixie/src/utils/shared/k8s"
+)
+
+// ClusterProvider uses whatever cluster your local kubeconfig points to.
+type ClusterProvider struct {
```

Comment:
shouldn't this be named `LocalClusterProvider`?
if you have a `cluster` package with a`Provider` interface, this struct is just one possible impl for the given interface.

Reply:
ah wait, this is in a `local` pacakge. nevermind. :)